kbase-extension/static/kbase/js/widgets/function_output/kbaseAttributeMapping.js

```diff
@@ -122,26 +115,32 @@ define([
                 columns: cols,
                 data: rows,
             };
-            const ConditionsTable = $tbl.DataTable(tblSettings);
-            //ConditionsTable.draw();
+            this.conditionsTable = $tbl.DataTable(tblSettings);
         },
 
         renderError: function (error) {
```

Comment:
This whole function could probably be supplanted with the ErrorDisplay module, but I'm already way out of scope here. Future work.